refinery/ui/source/js/dashboard/controllers/data-sets-card-ctrl.js

```diff
@@ -176,19 +179,32 @@
      * @param {string} query - user entered query for data set search
     **/
     function searchDataSets (query) {
+      // reset perm filter until we can search & check perms
+      vm.groupFilter = { selectedName: 'All' };
       if (query && query.length > 1) {
+        vm.loadingDataSets = true;
         var apiRequest = new DataSetSearchApi(query);
-        apiRequest().then(function (response) {
+        apiRequest(200).then(function (response) {
```

Comment:
Was there a similar success response that wasn't `200` here before, or is this just to be explicit?